bedrock/firefox/templates/firefox/tiles.html

```diff
@@ -45,197 +46,99 @@
+  </p>
 {% endif %}
+</section>
 
-<h3>{{ _('Respects your privacy') }}</h3>
-
-<p>
+<section class="tls-privacy">
 {% if LANG in ['en-US'] %}
-Tiles are a part of our initiative to set new standards for Internet advertisers and content publishers in the collection and use of personal data.
-</p>
-<p>
-Mozilla respects your privacy and any data collection by Mozilla via Tiles will be done in accordance with our <a href="{{ url('privacy.notices.firefox') }}">Privacy Notice</a>.
-The data we collect is to help us deliver the most interesting and relevant content to our users.
-For more on what data Mozilla collects via Tiles, see below.
+  <h2>{{ _('Personalization with your privacy in mind') }}</h2>
+  <p>{{ _('Mozilla promotes a transparent exchange between users and content providers to make the Web enjoyable and sustainable.') }}</p>
+  <p>{{ _('If suggestions are enabled, Firefox will make suggestions to you based on categories identified in your browsing history and will let you know every time why are you seeing specific suggestions.') }}</p>
+  <p>{{ _('In delivering Suggested sites, Firefox does not share your personal data, nor place cookies on your computer. Content providers will only receive aggregated information about interactions users have with their tiles, but individual information is never shared or sold.') }}</p>
+  <p>{{ _('You remain in control of your browsing history, and you can disable suggestions whenever your chose.') }}</p>
 {% else %}
-{{ _('As with all Mozilla products, Tiles provide users with extensive control over their personal data.') }}
-{{ _('Data collection is limited to the aggregate number of impressions, clicks and Tile placement.') }}
-{{ _('Nothing is ever personally identifiable. In other words, we don’t know who clicks on what.') }}
-{{ _('We just know the total number of views and clicks an individual Tile receives. This helps us deliver the most interesting and relevant content.') }}
-{{ _('You can deactivate click and impression tracking by clicking on the gear icon on the top right corner of the new tab page, and selecting Classic to show only unenhanced History Tiles or Blank mode.') }}
+  <h2>{{ _('Respects your privacy') }}</h2>
+  <p>
+    {{ _('As with all Mozilla products, Tiles provide users with extensive control over their personal data.') }}    
+    {{ _('Data collection is limited to the aggregate number of impressions, clicks and Tile placement.') }}   
+    {{ _('Nothing is ever personally identifiable. In other words, we don’t know who clicks on what.') }}    
+    {{ _('We just know the total number of views and clicks an individual Tile receives. This helps us deliver the most interesting and relevant content.') }}   
+    {{ _('You can deactivate click and impression tracking by clicking on the gear icon on the top right corner of the new tab page, and selecting Classic to show only unenhanced History Tiles or Blank mode.') }}   
+  </p>
```

Comment:
There are other instances like the above, I wonder whether we cannot just wrap the content inside a `{% trans %}` block, instead of each line separately? @flodolo